docs/build-ansible.rst

```diff
@@ -137,19 +145,19 @@ New minor releases (3.Y.0)
     rm -rf built
     mkdir built
 
-    # Create the ansible release
-    # (This generates a single tarball for ansible with a dep on the ansible-base package)
-    antsibull-build single 3.1.0 --data-dir ansible-build-data/3 --sdist-dir built --debian
+    # Prepare the ansible release
+    # (This generates the dependency file and updates changelog.yaml)
+    antsibull-build prepare 3.1.0 --data-dir ansible-build-data/3
 
     # Until we get separate versions for ansible-base working correctly:
     # https://github.com/ansible-community/antsibull/issues/187
-    # We'll need to update the ansible-base version manually and then rebuild the release. Follow
-    # these steps after running antsibull-build single above:
+    # We'll need to update the ansible-core version manually. Follow
+    # these steps after running antsibull-build prepare above:
     # vim ansible-build-data/3/ansible-3.1.0.deps
     # Change the ansible-base version information in here to the latest compatible version on pypi
 
-    rm -rf built
-    mkdir built
+    # Create the ansible release
+    # (This generates a single tarball for ansible with a dep on the ansible-base package)
```

Comment:
```suggestion
    # (This generates a single tarball for ansible with a dep on the ansible-core package)
```

Reply:
I think I'll do a more thourough `-base` to `-core` update once this is merged.